src/components/atoms/Checkbox.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+interface IProps {
+  className?: string;
+  id: string;
+  setState: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
@FredrikAugust I prefer a different naming convention for event handlers. In stead of passing down a prop called setState, call it onChange, onClick etc.

What are your thinking?

Reply:
Actually, this is not an event handler, so never mind. A more descriptive prop name is on my wish list.